lib/api.js

```diff
@@ -82,6 +82,58 @@ module.exports = {
 
         return node
       })
+  },
+  /**
+   * Error (Message)
+   *
+   * @memberof tree
+   * @method error
+   *
+   * @param  {String} plugin   Name ('posthtml-plugin')
+   * @param  {String} message  Error
+   *
+   * @return {Object} PostHTML Error
+   */
+  error: function (plugin, message) {
+    return this.messages.push({
+      type: 'error',
+      plugin: plugin,
+      message: message
+    })
+  },
+  /**
+   * Warning (Message)
+   *
+   * @memberof tree
+   * @method warning
+   *
+   * @param  {String} plugin   Name ('posthtml-plugin')
+   * @param  {String} message  Warning
+   *
+   * @return {Object} PostHTML Warning
+   */
+  warning: function (plugin, message) {
+    return this.messages.push({
+      type: 'warning',
+      plugin: plugin,
+      message: message
+    })
+  },
+  /**
+   * Dependency (Message)
+   *
+   * @memberof tree
+   * @method dependency
+   *
+   * @param  {String} file File
```

Comment:
👍 Yep the docs are a bit 'rough' atm 😛 , planned is to take a look at them altogether after features are set